mobilitypath_publisher/test/test_mobility_path_publisher.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include "mobilitypath_publisher/mobilitypath_publisher.hpp"
 
+/**
```

Comment:
Let's add the `These tests have been temporarily disabled ...` here as well

Reply:
added